.travis.yml

```diff
@@ -6,4 +6,5 @@ jdk:
 script: ./ext/travisci/test.sh
 notifications:
   email: false
-
+addons:
+  hostname: cljhttpclient
```

Comment:
Java needs to be able to resolve its own hostname, so I gave it a garbage name after the name of the project. There's a detailed discussion in this ticket about it: https://github.com/travis-ci/travis-ci/issues/5227#issuecomment-165131913

I can/will add that in a comment.